cargo build --features sse fails

### Problem
`cargo build --features sse` fails to compile on main (3e437ed56fad86001c4f955ac5d56bc9920de246)

### Workaround
Use feature `all`

### Detailed output
```
grzeg@DELL  D:\repos\hues   main ≢                                                                                                                                                                 [09:18]
❯ cargo build --features sse
  Downloaded futures-io v0.3.30
  Downloaded memchr v2.7.2
  Downloaded futures-macro v0.3.30
  Downloaded 3 crates (116.4 KB) in 0.41s
   Compiling cfg-if v1.0.0
   Compiling pin-project-lite v0.2.14
   Compiling itoa v1.0.11
   Compiling bytes v1.6.0
   Compiling scopeguard v1.2.0
   Compiling smallvec v1.13.2
   Compiling futures-core v0.3.30
   Compiling num_cpus v1.16.0
   Compiling once_cell v1.19.0
   Compiling fnv v1.0.7
   Compiling futures-task v0.3.30
   Compiling ryu v1.0.17
   Compiling equivalent v1.0.1
   Compiling tinyvec_macros v0.1.1
   Compiling hashbrown v0.14.3
   Compiling windows_x86_64_msvc v0.48.5
   Compiling windows_x86_64_msvc v0.52.5
   Compiling futures-sink v0.3.30
   Compiling pin-utils v0.1.0
   Compiling slab v0.4.9
   Compiling serde v1.0.198
   Compiling try-lock v0.2.5
   Compiling windows-targets v0.48.5                                                                                                                                                                            
   Compiling windows-targets v0.52.5
   Compiling lock_api v0.4.11
   Compiling futures-util v0.3.30
   Compiling tinyvec v1.6.0
   Compiling percent-encoding v2.3.1
   Compiling want v0.3.1
   Compiling futures-channel v0.3.30
   Compiling httparse v1.8.0                                                                                                                                                                                    
   Compiling tracing-core v0.1.32                                                                                                                                                                               
   Compiling tower-service v0.3.2
   Compiling httpdate v1.0.3
   Compiling unicode-bidi v0.3.15
   Compiling getrandom v0.2.14                                                                                                                                                                                  
   Compiling windows-sys v0.48.0
   Compiling windows-sys v0.52.0                                                                                                                                                                                
   Compiling parking_lot_core v0.9.9
   Compiling form_urlencoded v1.2.1
   Compiling base64 v0.21.7
   Compiling minimal-lexical v0.2.1
   Compiling memchr v2.7.2
   Compiling thiserror v1.0.59
   Compiling encoding_rs v0.8.34
   Compiling parking_lot v0.12.1                                                                                                                                                                                
   Compiling tracing v0.1.40
   Compiling http v0.2.12
   Compiling indexmap v2.2.6
   Compiling rustls-pemfile v1.0.4
   Compiling rand_core v0.6.4                                                                                                                                                                                   
   Compiling ipnet v2.9.0                                                                                                                                                                                       
   Compiling log v0.4.21                                                                                                                                                                                        
   Compiling unicode-normalization v0.1.23                                                                                                                                                                      
   Compiling nom v7.1.3                                                                                                                                                                                         
   Compiling ppv-lite86 v0.2.17
   Compiling mime v0.3.17
   Compiling sync_wrapper v0.1.2
   Compiling futures-timer v3.0.3
   Compiling http-body v0.4.6                                                                                                                                                                                   
   Compiling idna v0.5.0                                                                                                                                                                                        
   Compiling rand_chacha v0.3.1
   Compiling rand v0.8.5
   Compiling url v2.5.0
   Compiling serde_json v1.0.116
   Compiling serde_urlencoded v0.7.1
   Compiling eventsource-stream v0.2.3
   Compiling treediff v4.0.3
   Compiling json-patch v1.2.0
   Compiling mio v0.8.11
   Compiling winreg v0.50.0
   Compiling socket2 v0.5.6
   Compiling schannel v0.1.23
   Compiling tokio v1.37.0
   Compiling native-tls v0.2.11
   Compiling tokio-util v0.7.10
   Compiling tokio-native-tls v0.3.1
   Compiling h2 v0.3.26
   Compiling hyper v0.14.28
   Compiling hyper-tls v0.5.0
   Compiling reqwest v0.11.27
   Compiling reqwest-eventsource v0.5.0
   Compiling hues v0.1.2 (D:\repos\hues)
error[E0432]: unresolved import `futures_util`
   --> src\service\bridge.rs:152:17
    |
152 |             use futures_util::StreamExt;
    |                 ^^^^^^^^^^^^ use of undeclared crate or module `futures_util`

error[E0599]: no method named `next` found for struct `EventSource` in the current scope
   --> src\service\bridge.rs:157:48
    |
157 |                     while let Some(event) = es.next().await {
    |                                                ^^^^
    |
   ::: D:\packages\cargo\registry\src\index.crates.io-6f17d22bba15001f\futures-util-0.3.30\src\stream\stream\mod.rs:273:8
    |
273 |     fn next(&mut self) -> Next<'_, Self>
    |        ---- the method is available for `EventSource` here
    |
    = help: items from traits can only be used if the trait is in scope
help: trait `StreamExt` which provides `next` is implemented but not in scope; perhaps you want to import it
    |
1   + use futures_util::stream::stream::StreamExt;
    |
help: there is a method `try_next` with a similar name
    |
157 |                     while let Some(event) = es.try_next().await {
    |                                                ~~~~~~~~

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `hues` (lib) due to 2 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,7 +28,7 @@ required-features = ["mdns", "sse"]
 default = []
 all = ["mdns", "sse", "streaming"]
 mdns = ["dep:mdns", "dep:futures-util"]
-sse = ["dep:reqwest-eventsource"]
+sse = ["dep:reqwest-eventsource", "dep:futures-util"]
 streaming = ["dep:rustls", "dep:webrtc-dtls", "dep:webrtc-util"]
 
 [dependencies]

